docs/modules/ROOT/pages/install-clc.adoc

```diff
@@ -1,4 +1,133 @@
-= Installing Hazelcast CLC
-:description: The Hazelcast Command-Line Client (CLC) is available to install in macOS and Linux environments.
+= Installing the Hazelcast CLC
+:description: The Hazelcast Command-Line Client (CLC) is available to install in macOS, Linux, and Windows environments.
+
+// See https://docs.hazelcast.com/hazelcast/5.2-snapshot/clients/clc#installing-the-hazelcast-clc
+
+{description} You can install the Hazelcast CLC, using one of the following:
+
+* Homebrew (recommended)
+* Installer script
+* Building from source
+
+== Before You Begin
+
+See the following prerequisites for each installation option and operating system requirements.
+
+[tabs] 
+==== 
+Supported Operating Systems:: 
++ 
+-- 
+* Ubuntu 18.04 or newer
+* MacOS 12 or newer
+* Windows 10 or newer
+--
+
+Installer Script:: 
++ 
+-- 
+Installer script uses `curl` which is readily available on macOS computers.
+For Linux, follow the instructions https://everything.curl.dev/get/linux[here] if it is not already installed.
+--
+
+Homebrew::
++
+Follow the instructions https://docs.brew.sh/Installation[here] if it is not already installed on your macOS or Linux computer.
+
+Build from Source::
++
+If you prefer to install CLC by building it from its source on GitHub, you need to have the following installed on your machine:
+
+* https://www.atlassian.com/git/tutorials/install-git[Git]
+* https://www.gnu.org/software/make/[GNU Make]
+* https://go.dev/doc/install[Go v1.18] or newer
+====
+
+== Installing Hazelcast CLC
+
+You can install the Hazelcast CLC locally on macOS, Linux, and Windows environments.
+
+[tabs] 
+==== 
+Installer Script (macOS, Linux):: 
++ 
+-- 
+[source,bash]
+----
+curl https://raw.githubusercontent.com/hazelcast/hazelcast-commandline-client/main/scripts/install.sh | bash
+----
+--
+
+Installer (Windows)::
++
+The installer for Windows is located https://github.com/hazelcast/hazelcast-commandline-client/releases[here].
```

Comment:
Suggest that descriptive link text is used rather than [here] in all cases.

Reply:
Windows related content will be added once #92 is merged. Noted the comments, thank you Angela.